cli/up/build.go

```diff
@@ -451,42 +118,17 @@ func getPushedImages(images []string, authToken string) map[string]struct{} {
 				}
 
 				pushedImagesLock.Lock()
-				pushedImages[imgName] = struct{}{}
+				pushedImages[service] = imgName
```

Comment:
Does this need to have the digest? E.g. I'm imagining running blimp up after blimp build.

Reply:
Also what are the implications for existing users for changing the caching approach? Will their boot be slow? Will their images get rebuilt?

Reply:
> Does this need to have the digest?

No, it just uses the tag right now. If we want `blimp build && blimp up && blimp build && blimp up` to restart pods on the second blimp up, we should use the digest. I hadn't considered that.

> what are the implications for existing users for changing the caching approach?

We discussed this in standup, I'm adding detection of old-style blimp cache so everything does not have to be rebuilt.

Reply:
I think we need to support the `blimp up && blimp build && blimp up` workflow. It's what we've recommended to kustomer